server/views/pages/allCompletedReports.njk

```diff
@@ -50,15 +63,14 @@
     <div class="results-table">
       {{ govukTable({
       head: [
-        { html: 'Prisoner&#8217;s name' },
-        { text: "Prison number" },
-        { text: "Date and time of incident",
-          attributes: {
+        { text: "Discovery date and time",
+                  attributes: {
           "aria-sort": "descending"
         }
         },
-        { text: "Reporting officer" },
+        { html: "Name and prison number" },
         { text: "Status" },
+        { text: "Latest scheduled hearing" },
         { text: "" }
       ],
```

Comment:
you need another `{ text: "" }` here for the final header space (otherwise the table header is too short